app-modules/student-data-model/src/Filament/Resources/StudentResource/RelationManagers/StudentFormSubmissionsRelationManager.php

```diff
@@ -0,0 +1,119 @@
+use AdvisingApp\Form\Models\FormSubmission;
+use Filament\Tables\Actions\BulkActionGroup;
+use Filament\Tables\Actions\DeleteBulkAction;
+use AdvisingApp\Form\Enums\FormSubmissionStatus;
+use AdvisingApp\Form\Filament\Resources\FormResource;
+use Filament\Resources\RelationManagers\RelationManager;
+use AdvisingApp\Form\Filament\Actions\RequestFormSubmission;
+use AdvisingApp\Form\Filament\Tables\Filters\FormSubmissionStatusFilter;
+
+class StudentFormSubmissionsRelationManager extends RelationManager
+{
+    protected static string $relationship = 'formSubmissions';
+
+    protected static ?string $title = 'Forms';
+
+    public static function canViewForRecord(Model $ownerRecord, string $pageClass): bool
+    {
+        return Gate::check(Feature::OnlineForms->getGateName());
+    }
+
+    public function table(Table $table): Table
+    {
+        return $table
+            ->columns([
+                IdColumn::make(),
+                TextColumn::make('submissible.name')
+                    ->label('Form')
+                    ->searchable()
+                    ->url(fn (FormSubmission $record): string => FormResource::getUrl('edit', ['record' => $record->submissible])),
+                TextColumn::make('status')
+                    ->badge()
+                    ->getStateUsing(fn (FormSubmission $record): FormSubmissionStatus => $record->getStatus()),
+                TextColumn::make('submitted_at')
+                    ->dateTime()
+                    ->sortable(),
+                TextColumn::make('requester.name'),
+                TextColumn::make('requested_at')
+                    ->dateTime()
+                    ->getStateUsing(fn (FormSubmission $record): ?CarbonInterface => $record->requester ? $record->created_at : null),
+            ])
+            ->filters([
+                FormSubmissionStatusFilter::make(),
+            ])
+            ->headerActions([
+                RequestFormSubmission::make(),
+            ])
+            ->actions([
+                ViewAction::make()
+                    ->modalHeading(fn (FormSubmission $record) => 'Submission Details: ' . $record->submitted_at->format('M j, Y H:i:s'))
+                    ->modalContent(fn (FormSubmission $record) => view('form::submission', ['submission' => $record]))
+                    ->visible(fn (FormSubmission $record) => $record->submitted_at),
+                DeleteAction::make(),
+            ])
+            ->bulkActions([
+                BulkActionGroup::make([
+                    DeleteBulkAction::make(),
+                ]),
+            ]);
+    }
+
+    public function getTableRecordKey(Model $record): string
+    {
+        return base64_encode(json_encode($record->attributesToArray()));
+    }
+
+    protected function resolveTableRecord(?string $key): ?Model
+    {
+        return $this->getTable()->getQuery()
+            ->where(json_decode(base64_decode($key), associative: true))
+            ->first();
+    }
```

Comment:
These can be removed. They are not needed and would make this less performant as these records have primary keys.